tests/test_validation.py

```diff
@@ -348,6 +348,37 @@ def test_string_too_long_causes_validation_error():
     assert "under_character_limit" in errs['serviceName']
 
 
+def test_price_not_money_format_validation_error():
+    cases = [
+        "",  # not provided
+        "foo",  # not numeric
+        "12.1",  # too few decimal places
+        "12.001",  # too many decimal places
```

Comment:
Lots of decimal places is OK if your price is less than £1, eg £0.005 per gigabyte is a reasonable figure for some cloud storage.

Refer to: https://github.com/alphagov/supplier-submission-portal/commit/6a8c5003a434cccfdc5dfa4937d6dfefae3ebc10